src/pages/components/tooltip/usage.mdx

```diff
@@ -123,115 +208,210 @@ import { Filter16 } from '@carbon/icons-react';
-may be aligned to **start**, **center** or **end**.
+## Formatting
 
-Note that left and right positioning is not available for
-[definition tooltip](#definition-tooltip). This ensures the tooltip does not
-obstruct important information to the left or right of the trigger word.
+### Standard tooltip anatomy
 
 <Row>
 <Column colLg={8}>
 
-![Tooltips positioned at top](images/tooltip-usage-1-top.png)
+![Image of tooltip that has a UI trigger button paired with a container](images/tooltip-usage-5.png)
 
 </Column>
 </Row>
 
+1. **UI trigger button:** Element that triggers a tooltip on hover or focus
+2. **Caret tip:** Closely associates container to specific trigger element
+3. **Container:** Contains helper text
+
+### Icon button tooltip anatomy
+
 <Row>
 <Column colLg={8}>
 
-![Tooltips positioned to the left and right.](images/tooltip-usage-1-left-right.png)
+![Image of tooltip that has an icon button paired with a container](images/tooltip-usage-6.png)
 
 </Column>
 </Row>
 
+1. **Icon button:** Button that triggers a tooltip on hover or focus
+2. **Caret tip:** Closely associates container to specific trigger element
+3. **Container:** Contains helper text
+
+### Alignment
+
+The container of the tooltip may be aligned to **start**, **center** or **end**
+to keep the container from bleeding off the page or covering important
+information. The UI trigger button and caret tip should be vertically center
+with each other to clearly associate the tooltip and the trigger. This is
+especially helpful when multiple elements are close to each other.
+
 <Row>
 <Column colLg={8}>
 
-![Tooltips positioned at the bottom](images/tooltip-usage-1-bottom.png)
+![Image of tooltips aligned to start, center, and end](images/tooltip-usage-7.png)
 
 </Column>
 </Row>
 
-## Variants
-
-Tooltips provide additional, contextual information. Each variant achieves this
-for different design needs.
+<Caption>Tooltip containers may be aligned to start, center or end</Caption>
 
-### Icon tooltip
+### Placement
 
-An icon tooltip is used to clarify the action or name of an interactive icon
-button.
+Tooltip directions by default are set to auto. Upon opening, tooltips can detect
+the edges of the browser to properly be placed in view so the container does not
+get cutoff. Tooltips can instead use specific directions and may be
+positioned **right**, **left**, **bottom**, or **top** to the trigger item. Do
+not cover related content that is essential to the user's tasks. Tooltips should
+not bleed off page or behind other content.
 
 <Row>
 <Column colLg={8}>
 
-![Example image of an icon tooltip.](images/tooltip-usage-1.png)
+![Image of tooltips positioned to the right, left, bottom, or top](images/tooltip-usage-7.png)
 
 </Column>
 </Row>
 
-#### Guidance
+<Caption>
+  Tooltip containers can be positioned to the right, left, bottom, or top.
+</Caption>
+
+## Content
+
+### Main elements
+
+#### Text
```

Comment:
I'm actually not sure what the rule is for this but some on the points here end when a period and some of them don't have them. I'm wondering if they all should be consistent? Not sure though which way or if they can be mixed.

Reply:
From a quick review of other pages, they all have periods at the end.